### PR TITLE
ADEN-7802 Adding Pubmatic to FV

### PR DESCRIPTION
**Pull request by Bielik20**

### CODE REVIEW

**Comment by dmnsobczak** [comment]
Major due to Prebid update: 1.16 >>> 1.31

**Comment by dmnsobczak** [comment]
We want to serve Pubmatic Video through DFP. To do that we need to pass `hb_uuid` parameter like for AppNexusAst:
https://github.com/Wikia/ad-engine/blob/dev/src/ad-bidders/prebid/prebid-settings.js#L37
Also we can add Instant Global, just like for AppNexusAst, to rollout this feature granually:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/context/adContext.js#L113
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/src/ad-engine.bridge.js#L114

**Comment by dmnsobczak** [comment]
Unit tests and linter are failing: please fix them :)

File: spec/ad-bidders/prebid/prebid-settings.spec.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { expect } from 'chai';
+import { context } from '../../../src/ad-engine/services/context-service';
+import { getSettings } from '../../../src/ad-bidders/prebid/prebid-settings';
+
+describe('getBidderUuid', () => {
```
Comment (dmnsobczak) [inline spec/ad-bidders/prebid/prebid-settings.spec.js]: 💎

**Comment by jacekbj** [comment]
It looks like we need to change userSync settings:
http://prebid.org/dev-docs/bidders/pubmatic.html (search for "For Prebid.js v1.15.0 and later:")
https://github.com/Wikia/ad-engine/blob/041800d5c35e4ea8ad5044181b5f206482c6857d/src/ad-bidders/prebid/index.js#L39
@dmnsobczak @fraszczakszymon

**Pull request merged**